src/utils/accounts.js

```diff
@@ -1,11 +1,13 @@
 import { v4 as uuidv4 } from 'uuid'
 import { getAssetIcon } from '@/utils/asset'
+import { getDerivationPath } from '@/utils/derivationPath'
 import { chains } from '@liquality/cryptoassets'
 
 export const accountCreator = (payload) => {
-  const { walletId, account } = payload
+  const { network, walletId, account } = payload
   const {
     name,
+    alias,
```

Comment:
What is the deal with this `alias` thing? I see it's been added to all the structures, but I can't see where it's being used.

Reply:
it's related to the structure that we set with ui/ux, for account names we will show something like:
`{name} - {alias}`, an example can be `Bictoin 1 - My user account name`